packages/plugins/users-permissions/documentation/content-api.yaml

```diff
@@ -273,8 +272,7 @@ paths:
                   email:
                     type: string
                   sent:
-                    type: enum
-                    enum: [true]
+                    type: boolean
```

Comment:
this is invalid the use of enum here is because the value can only be true not just a boolean. it the same for all the places where with used this method.

Reply:
Ok, I understand,  but it is quite correct than too... Because here Swagger/OAS3 asks for the type of the field not the value... I've checked it in Postman, there is returned a boolean=true and not a string='true'! So the endpoint returns a boolean (true or falls) but it only returns true, which can't define more specific in OAS as far as I know... If it should be an enum anyways the correct syntax is: 
```
    type: string
    enum: [true]
```
or
```
    type: string
    enum: 
        - true
```

Reply:
Feel free to update your PR to relect this 👍